### ISSUE TITLE
Error when using PIXI.prepare with Graphics

### ISSUE THREAD

**Issue by Ajyress** [pixijs/pixi.js#6122]
Hi,

I'm trying to reproduce the example given here using Pixi V5:
https://pixijs.download/dev/docs/PIXI.prepare.html

Here is my code:

```
pixiApp.stop();

const rect = new Graphics();
rect.beginFill(0x00ff00);
rect.drawRect(40, 40, 200, 200);

pixiApp.stage.addChild(rect)

pixiApp.renderer.plugins.prepare.upload(pixiApp.stage, () => {
    pixiApp.start();
});
```

### Expected Behavior

Should work

### Current Behavior

I get the following error in browser console:

> Uncaught TypeError: Cannot read property 'CONTEXT_UID' of undefined
    at Array.uploadGraphics (prepare.es.js:620)
    at Prepare.prepareItems (prepare.es.js:247)
    at BasePrepare.delayedTick (prepare.es.js:160)
    at ZoneDelegate.invokeTask (zone-evergreen.js:391)
    at Zone.runTask (zone-evergreen.js:168)
    at invokeTask (zone-evergreen.js:465)
    at ZoneTask.invoke (zone-evergreen.js:454)
    at timer (zone-evergreen.js:2650)

### Environment

- **`pixi.js` version**: _5.1.3_
- **Browser & Version**: _Chrome OS 77.0.3865.63 (Build officiel) beta (64 bits)_

**Comment by kossovec** [comment]
I have the same issue.

**Comment by themoonrat** [comment]
@kossovec  Could you try the dev version please? I believe https://github.com/pixijs/pixi.js/pull/6193 will have fixed the issue

**Comment by kossovec** [comment]
@themoonrat  Fixed in 5.2.0 thx.
